### PR TITLE
Add www.grotto-bestiary.com

### PR DESCRIPTION
**Pull request by 6TELOIV**
This is a cool list, and I would love for my site to be included! The source code can be found at https://github.com/6TELOIV/grotto-bestiary/

I just made it open sourced, so I will be working on documenting how to set up a development environment etc soon.

### CODE REVIEW

**Comment by stabswell** [comment]
Hi @6TELOIV the project looks cool af. I fixed the title for you, lemme know if thats alright

@s-crypt can we add this project to the site? I was thinking I'd be nice to create a Grotto Beast section aswell

**Comment by s-crypt** [comment]
@stabswell yeah if the others think its a good idea, go for it! If it does get added, I think a grotto beasts section under the main section would be perfect

**Comment by s-crypt** [comment]
@stabswell I am not so sure about this one. We have already had to take down a site because it used assets without permission. Seeing as the site has scans of all the cards, I am apprehensive to include this because it may not be approved by the creators of grotto beasts (and infringe on intellectual property rights).

**Comment by 6TELOIV** [comment]
Title sounds good @stabswell 

@s-crypt I need to check again, but I have gotten a soft OK from Jerma and DoodleBit on the site (Something along the lines of "We'll create community guidelines soon, but please make stuff like this")

They have not created community guidelines yet, so can't get anything official.

I'll try to search down their discord messages regarding this

**Comment by 6TELOIV** [comment]
Jerma on fan sites, specifically discussing this site.
image

DoodleBit's only issue he's mentioned with the site (the unofficial TTS mod exporter) which will be removed from the site once the official mod goes live
image

DoodleBit follow up on above, with TTS mod getting near to completion
image

Most recent follow up with DoodleBit regarding Fan Content Guidelines
image

As one other aside, my site has been approved for linking within the GB discord channel by the mods
image

---

Summing up my experience so far, they seem to be showing soft support of the site, but are waiting for their official fan-works guidelines before approving of anything. Once they release the guideline, I'll be sure to comply with it, and if they are against me hosting the card art, I'll remove them and switch over to the images from the wiki whenever it goes live as long as they're OK with that.

**Comment by sticks-stuff** [comment]
@6TELOIV do you have a discord so we can add you to the jermasites discord?

**Comment by 6TELOIV** [comment]
@sticks-stuff Yes, same as my github username: `@6teloiv`
